sdk/translation/ai-translation-document-rest/README.md

```diff
@@ -0,0 +1,346 @@
+# DocumentTranslation REST client library for JavaScript
+
+Document Translation is a cloud-based machine translation feature of the Azure AI Translator service. You can translate multiple and complex documents across all supported languages and dialects while preserving original document structure and data format. The Document translation API supports two translation processes:
+
+Asynchronous batch translation supports the processing of multiple documents and large files. The batch translation process requires an Azure Blob storage account with storage containers for your source and translated documents.
+
+Synchronous single file supports the processing of single file translations. The file translation process doesn't require an Azure Blob storage account. The final response contains the translated document and is returned directly to the calling client.
+
+The following operations are supported by the Document Translation feature:
+
+Synchronous document translation: Used to synchronously translate a single document. The method doesn't require an Azure Blob storage account.
```

Comment:
nitpick: use a bullet list of features for better readability?
```suggestion
- **Synchronous document translation**: Used to synchronously translate a single document. The method doesn't require an Azure Blob storage account.
```

Reply:
Thanks for the suggestion, Updated